shared/constants/permissions.ts

```diff
@@ -21,6 +21,12 @@ export const RestrictedMethods = Object.freeze({
   ///: END:ONLY_INCLUDE_IF
 } as const);
 
+///: BEGIN:ONLY_INCLUDE_IF(snaps)
+export const ConnectionPermission = Object.freeze({
```

Comment:
Document that this isn't really a permission

Reply:
Added comment.